### ISSUE TITLE
Using ADC for encoding? 

### ISSUE THREAD

**Issue by rog77** [hrvach/talkberry#1]
Do you have any plans to use the onboard ADC for encoding? I think this project could make you quite popular in ham radio circles - one would be able to easily modify a fist mic to a cheap HT with your work, and upgrade an inexpensive radio to DV for less than $10.

**Comment by hrvach** [comment]
Yes, that's one direction I'll try - I already experimented with is using a cheap PDM MEMS microphone, but ADC means less parts and lower price. With 12-bit resolution it should be sufficient to provide acceptable quality.

**Comment by rog77** [comment]
Thanks for the response, that sounds positive :-)
